### PR TITLE
Rhev vms tab

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

**Comment by lhellebr** [comment]
https://github.com/SatelliteQE/robottelo/pull/6076/commits/7beb510fcf675b958a36dedf0eac24b702c4dd8f depends on this

File: airgun/entities/computeresource.py [hunk 1]
```diff
@@ -33,6 +34,12 @@ def delete(self, value):
         view.table.row(name=value)['Actions'].widget.fill('Delete')
         self.browser.handle_alert()
 
+    def list_RHEV_VMs(self, rhev_name, expected_vm_name):
```
Comment (abalakh) [inline airgun/entities/computeresource.py]: 1) why not `list_vms`? and in any case - use lowercase.
2) make expected_vm_name optional (`expected_vm_name=None`)
3) pls add docstring
4) `else view.virtual_machines.table.row()` why just single row() and not rows()? Helpers is called list VMs, not just 1 vm.

File: airgun/views/computeresource.py [hunk 2]
```diff
@@ -6,8 +6,10 @@
     View,
     ConditionalSwitchableView,
 )
-from airgun.views.common import BaseLoggedInView, SearchableViewMixin
-from airgun.widgets import FilteredDropdown, ActionsDropdown, SatTable
+from airgun.views.common import BaseLoggedInView, \
+        SearchableViewMixin, SatTab, SatTable
```
Comment (abalakh) [inline airgun/views/computeresource.py]: SatTable isn't a view, flake warning was actually right :) Pls revert this change and import it from `airgun.widgets` instead.
And btw try to avoid backslashes and use following format instead
```python
from airgun.views.common import (
    BaseLoggedInView,
    SearchableViewMixin,
    SatTab,
)
```
if it doesn't fit in 1 line

File: airgun/views/computeresource.py [hunk 3]
```diff
@@ -112,3 +114,28 @@ def after_fill(self, was_change):
...
+    breadcrumb = BreadCrumb()
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Compute resources'
+                and not self.breadcrumb.read().starts_with('Edit ')
```
Comment (abalakh) [inline airgun/views/computeresource.py]: Maybe there's smth a bit better to count on, like `details` word in current location or smth similar? If not - then, at least, add one more condition `and not self.breadcrumb.read() == 'Create Compute Resource'` but we'd rather specify what should be there instead everything that shouldn't :)

File: airgun/views/computeresource.py [hunk 4]
```diff
@@ -112,3 +114,28 @@ def after_fill(self, was_change):
...
+        )
+
+    @View.nested
+    class virtual_machines(SatTab):
+        TAB_NAME = 'Virtual Machines'
+
+        table = SatTable(
```
Comment (abalakh) [inline airgun/views/computeresource.py]: nitpick: this should fit in one line

File: airgun/views/organization.py [hunk 5]
```diff
@@ -156,4 +156,4 @@ class locations(SatVerticalTab):
 
     @View.nested
     class parameters(SatVerticalTab):
-        resources = CustomParameter()
+        resources = CustomParameter(id='global_parameters_table')
```
Comment (abalakh) [inline airgun/views/organization.py]: this change is already merged. could you please rebase your branch?

**Comment by lhellebr** [comment]
Good points, thanks for feedback. Everything should be fixed now.

**Comment by lhellebr** [comment]
```
$ pytest tests/foreman/ui_airgun/test_computeresource.py::test_positive_v3_wui_virtual_machines_get_loaded
======================================= test session starts ========================================
platform linux -- Python 3.6.5, pytest-3.6.1, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/lhellebr/git/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collecting 1 item                                                                                  2018-06-21 13:54:26 - conftest - DEBUG - Collected 1 test cases

collected 1 item                                                                                   

tests/foreman/ui_airgun/test_computeresource.py .                                             [100%]

===================================== 1 passed in 56.04 seconds =====================================
```

**Comment by jhutar** [comment]
To unblock Robottelo tests PR waiting for this. Please let me know if I need to do something differently next time.
